### PR TITLE
Created footer layout

### LINKED ISSUE #18: Create a footer component

### PR DESCRIPTION
**Pull request by Lunchii**
closes #18 
## Overview
Three icons (email, instagram, discord).
Service times (sundays and fridays)
probably doesn't work with mobile and odd behaviour with page zooming (needs a fix)

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const geistMono = Geist_Mono({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
+  title: "LKPC EM",
```
Comment (Cheekane) [inline app/layout.tsx:16]: Make this `LondonKPC EM`

**Review by Cheekane — CHANGES_REQUESTED** [review]
Can you make the service schedule only show Sunday's time.

File: components/Footer.tsx [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
+        </div>
+      </div>
+      <div className="flex w-full bg-[#18181a] py-14">
+        <div className="px-8 text-[30px] flex-1 text-center">
+          LondonKPC
+        </div>
+        <div className="px-8 text-[30px] flex-1 text-center">
+          <p>Service</p>
+          <div className="text-[16px]">
+            <p>Sunday: 2:15pm</p>
```
Comment (jerpyk) [inline components/Footer.tsx:29]: adding end time would be good

File: app/home/page.tsx [hunk 3]
```diff
@@ -1,21 +1,25 @@
...
-            Home...
+    <div className="h-screen w-full overflow-x-hidden flex flex-col">
+      <div className="flex flex-1">
+        <div className="w-1/2 bg-slate-700 text-center opacity-90 hover:opacity-100 flex items-center justify-center">
+          Home...
+        </div>
+        <div className="w-1/2">
+          <img
+            src={capy.src}
+            alt="capybara"
```
Comment (jerpyk) [inline app/home/page.tsx:15]: nice

**Review by Cheekane — CHANGES_REQUESTED** [review]
Mobile view doesn't work.

**Pull request closed without merging**